tests/e2e_tests/client_utils.py

```
@@ -38,6 +38,25 @@ def wait_for_server(self, timeout: int = 30) -> bool:
+        deadline = time.time() + timeout
+        while time.time() < deadline:
+            try:
+                status = self.client.cluster_status()
+                if (getattr(status, "status", None) == "enabled"
+                        and len(status.peers) >= expected_peers
+                        and status.raft_info.leader is not None
+                        and status.raft_info.pending_operations == 0):
+                    print(f"Cluster ready with {len(status.peers)} peers, leader={status.raft_info.leader}")
+                    return True
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)